### ISSUE TITLE
Update of Samba

### ISSUE THREAD

**Issue by Neustradamus** [zentyal/zentyal#2014]
Can you update Samba?
- https://www.samba.org/
- https://www.samba.org/samba/latest_news.html

L.P.H. van Belle is an important and serious Samba guy with quality, you can look on samba ML
- https://lists.samba.org/archive/samba/

Explanation for repos:
- https://lists.samba.org/archive/samba/2020-September/232145.html

Links:
- https://www.van-belle.nl/
- http://apt.van-belle.nl/
- https://apt.van-belle.nl/debian/dists/
- https://apt.van-belle.nl/debian/dists/focal-samba412/
- https://apt.van-belle.nl/debian/dists/focal-samba413/

Thanks in advance.

Linked to:
- https://github.com/zentyal/zentyal/issues/2008

**Comment by ajh123** [comment]
Yes, this would be nice, especially Samba 4.16. Currently Windows 11 cannot join a domain with Zentyal 7.1.
